data/society/public-safety/index.html

```diff
@@ -60,6 +60,7 @@
     name: Utah Law Enforcement Boundaries
     item_id: 2e29c65cc23c48bd88b619a3ea218626
   updates:
+    - October 11, 2023 - Updates based on recent municipal annexations
```

Comment:
**polish**: we try to only keep 10 updates. do you mind trimming some of the earlier ones?

Reply:
yup, for sure.